Cargo.toml

```diff
@@ -1943,6 +1943,28 @@ description = "Creates a hierarchy of parents and children entities"
 category = "ECS (Entity Component System)"
 wasm = false
 
+[[example]]
+name = "immutable_components_dynamic"
+path = "examples/ecs/immutable_components_dynamic.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.immutable_components_dynamic]
+name = "Immutable Dynamic Components"
+description = "Demonstrates the creation and utility of dynamic immutable components"
+category = "ECS (Entity Component System)"
+wasm = false
+
+[[example]]
+name = "immutable_components"
```

Comment:
Capitalization inconsistency with the names

Reply:
I'm not sure what you're referring to here sorry. Comparing to the `iter_combinations` example:

```toml
[[example]]
name = "iter_combinations"
path = "examples/ecs/iter_combinations.rs"
doc-scrape-examples = true

[package.metadata.example.iter_combinations]
name = "Iter Combinations"
description = "Shows how to iterate over combinations of query results"
category = "ECS (Entity Component System)"
wasm = true
```

I believe I'm matching the same lower_snake_case for `example.name`, and Title Case for `package.metadata.example.x.name`.

Reply:
Sorry, I wasn't paying enough attention.

Reply:
All good! Just confused haha